From d569dbb32fe10c067a48af269b3abe0f901f4735 Mon Sep 17 00:00:00 2001
From: Grace Cham <hscham@chromium.org>
Date: Wed, 2 Nov 2022 10:56:06 +0900
Subject: [PATCH] backward compatibility: do not use None for typename

Conflict with the None and Bool macros in X11/X.h used in client
packages.

Change-Id: Ic3b650abdefcc87abc48e84b5dbfada88602759a
---
 base/cxx20_to_address.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/base/cxx20_to_address.h b/base/cxx20_to_address.h
index 0010aecd4080..a1641146f050 100644
--- a/base/cxx20_to_address.h
+++ b/base/cxx20_to_address.h
@@ -28,8 +28,8 @@ constexpr auto to_address(const Ptr& p) noexcept
   return std::pointer_traits<Ptr>::to_address(p);
 }
 
-template <typename Ptr, typename... None>
-constexpr auto to_address(const Ptr& p, None...) noexcept {
+template <typename Ptr, typename... Args>
+constexpr auto to_address(const Ptr& p, Args...) noexcept {
   return base::to_address(p.operator->());
 }
 
-- 
2.38.1.273.g43a17bfeac-goog

